packages/catala-format/catala-format.0.2.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "vincent.botbol@inria.fr"
+authors: [ "Vincent Botbol" ]
+
+homepage: "https://github.com/CatalaLang/catala-format"
+bug-reports: "https://github.com/CatalaLang/catala-format"
+dev-repo: "git+https://github.com/CatalaLang/catala-format.git"
+
+license: "Apache-2.0"
+depends: [
+  "ocaml"
+  "dune"
+  "topiary" {= "0.5.1"}
```

Comment:
```suggestion
  "cmdliner"
  "lwt" {with-test}
  "topiary" {= "0.5.1"}
```

@vincent-botbol please update the upstream opam file with the missing dependencies